crane/container.go

```diff
@@ -309,6 +311,10 @@ func (b BuildParameters) File() string {
 	return expandEnv(b.RawFile)
 }
 
+func (b BuildParameters) Args() []string {
```

Comment:
`BuildArgs` ? it's redundant but so is the docker flag name, so it's better to be consistent IMHO